man/dnf.plugin.kpatch.8

```diff
@@ -0,0 +1,34 @@
+.TH man 8 "23 Apr 2020" "1.0" "dnf-kpatch man page"
+.SH NAME
+dnf\-kpatch \- kpatch patch module installation plugin
+
+.SH SYNOPSIS
+dnf kpatch <install\-mode>
+
+.SH DESCRIPTION
+
+dnf\-kpatch is a dnf plugin providing automated installation of kpatch\-patch
+packages. When enabled, installation of new kernel packages via dnf will trigger
+installation of the latest kpatch-patch package targeting the same kernel version.
+
+
+.SH INSTALL MODES
+
+.B auto
+       Installs missing kpatch\-patch packages for installed kernels and enables
+       automatic kpatch-patch installation for future kernel installations.
+
+.B manual
+       Disables automatic installation of kpatch-patch pacakges.
+
+
+.SH SEE ALSO
+dnf(1) kpatch (1)
+
+.SH BUGS
+No known bugs.
+
+.SH AUTHOR
+Julien Thierry <jthierry@redhat.com>
```

Comment:
`.UR mailto:jthierry@redhat.com`
`Julien Thierry`
`.UE`

Reply:
This gives a weird output in the man:
`Julien Thierry (mailto:jthierry@redhat.com)`

Is that intended?

Reply:
Yes; it's, however, a matter of personal preference (unless there's a project-wide policy in place) on how to format (e-mail) links; to my taste, `.UR` macro is more "semantic".

Reply:
I see, I'll keep your suggestion and see if the rest of the team is happy with it or has other taste :)